packages/compass-components/src/components/accordion.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { spacing } from '@leafygreen-ui/tokens';
+import { css } from '@emotion/css';
+import Icon from '@leafygreen-ui/icon';
+import { uiColors } from '@leafygreen-ui/palette';
+import { useId } from '@react-aria/utils';
+
+const labelStyles = css({
+  padding: 0,
+  margin: 0,
+  fontWeight: 'bold',
+});
+
+const buttonStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+  borderStyle: 'solid',
+  borderColor: 'transparent',
+  borderRadius: '6px',
+  borderWidth: '3px',
+  background: 'none',
+  '&:focus-visible': {
+    borderColor: uiColors.green.base,
+  },
```

Comment:
Not sure if this was discussed before, but this seems to be different from what styles leafygreen is using for their focus outline (they even export a special focus color from the palette), I'm wondering if we want to keep compat with them so that when we are mixing leafygreen and our components on the screen, they look the same, this is how it can be done (I switched this suggestion to use box-shadow because this way it's also easier to animate without worrying about element shifting around due to the bounding box size changes, this is also similar to what leafygreen components are doing):

```suggestion
  border: 'none',
  background: 'none',
  borderRadius: '6px', 
  boxShadow: 'none',
  transition: 'box-shadow 150ms ease-in-out',
  '&:focus-visible': {
    boxShadow: `0 0 0 3px ${uiColors.focus}`,
  },
```

What do you think?

Reply:
I spoke with Claudia about that, we were not sure about the color to use, but I guess if we have a `uiColors.focus` exported we should use that. :+1: